documentation/src/test/java/example/ParameterizedTestDemo.java

```diff
@@ -135,9 +136,9 @@ void testWithMultiArgMethodSource(String first, int second) {
 
 	// tag::CsvSource_example[]
 	@ParameterizedTest
-	@CsvSource({ "foo, 1", "bar, 2", "'baz, qux', 3" })
+	@CsvSource({ "foo, 1", "bar, 2", "'baz, qux', 3", "'', 4", ",5" })
 	void testWithCsvSource(String first, int second) {
-		assertNotNull(first);
+		assertNotNull(Objects.toString(first, "<null>"));
```

Comment:
Sure. It's _difficult_ because it's a meaningless demo (like many of our demos). 😉 

So perhaps it would be better and clearer to the user to move the empty and null use cases to separate test methods.

If we don't do that, I'd recommend making use of some conditional logic for the assertion performed for the first parameter, but... I think that makes the example overly complicated. Thus, I still prefer separate test methods.